conda_index/index/__init__.py

```diff
@@ -448,9 +452,10 @@ def _build_current_repodata(subdir, repodata, pins):
     for keep_pkg in keep_pkgs:
         if keep_pkg.fn.endswith(CONDA_PACKAGE_EXTENSION_V2):
             conda_packages[keep_pkg.fn] = repodata["packages.conda"][keep_pkg.fn]
-            # in order to prevent package churn we consider the md5 for the .tar.bz2 that matches the .conda file
-            #    This holds when .conda files contain the same files as .tar.bz2, which is an assumption we'll make
-            #    until it becomes more prevalent that people provide only .conda files and just skip .tar.bz2
+            # In order to prevent package churn, we consider the md5 for the .tar.bz2 that matches
+            # the .conda file.  This holds true when .conda files contain the same files as
+            # .tar.bz2, which is an assumption we'll make until it becomes more prevalent that
+            # people provide only .conda files and just skip .tar.bz2.
```

Comment:
We should check this again now that we use our _speed_ to index `.conda` and `.tar.bz2` independently.